src/pages/overview/document-generation-api/tablewithmarkers.md

```diff
@@ -0,0 +1,335 @@
+
+**Note:** discard-if construct can also take a context input as discard-if(expr(**condition**),**context_tag**). In this case the `condition`
+will be evaluated in the context of to `context_tag`.
+
+JSON representation of the input data:
+
+```json
+{
+  "Conversion": [
+    {
+      "Month": "July, 2021",
+      "Rate": 10,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "September, 2021",
+      "Rate": 30,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "Dec, 2021",
+      "Rate": 20,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "April, 2022",
+      "Rate": 20,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "Dec, 2022",
+      "Rate": 30,
+      "Method": "Payout by Cheque"
+    }
+  ]
+}
+```
+
+
+![Discard if with table markers](../images/discard_if_table_markers.png)
+
+In the above example we've used discard-if along with context **$**($ here indicates that the context is the input json itself). 
+So, in this case the conditions will be evaluated in the context of input json.
+
+## Dynamic table rows
+Discard a row or set of rows in a table from the final generated document.
+
+### Discard row if condition evaluates to true
+A row in the table can be discarded only when the condition in the discard-row-if(expr(**condition**)) evaluates
+to true in the provided context. Add this construct along with the template tag to discard the row based on the provided condition.
+
+For Example: {{PROPERTY:**discard-row-if(expr(condition))**}}
+
+**Note:** We can also use context in a similar way to discard-if, as {{PROPERTY:**discard-row-if(expr(condition),context_tag)**}}  .
+
+JSON representation of the input data:
+```json
+{
+  "Conversion": [
+    {
+      "Month": "July, 2021",
+      "Rate": 10,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "Dec, 2021",
+      "Rate": 20,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "Dec, 2022",
+      "Rate": 40,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "Dec, 2021",
+      "Rate": 80,
+      "Method": "Payout by Cheque"
+    },
+    {
+      "Month": "April, 2022",
+      "Rate": 100,
+      "Method": "Payout by Cash"
+    },
+    {
+      "Month": "Dec, 2022",
+      "Rate": 45,
+      "Method": "Payout by Cheque"
+    }
+  ]
+}
+```
+
+
+![Discard row if with table markers](../images/discard_row_if_table_markers.png)
+
+## Conditions inside tables
+Conditions can be used inside tables with table markers.
+
+JSON representation of the input data:
+
+```json
+{
+  "company": [
+    {
+      "name": "ABC",
+      "supportsX" : "TRUE"
+    },
+    {
+      "name": "XYZ",
+      "supportsX": "FALSE"
+    }
+
+  ]
+}
+```
+
+![Conditions inside tables with markers](../images/conditions_inside_tables_table_markers.png)
+
+## Nested Tables
+Table tags with markers can also be used inside another table.
```

Comment:
Nested Tables or Table inside Table is **only** supported using the new Table Markers ("codified"......)

Reply:
Nested Table should contain its own Table Markers as per the rules defined above -> provide the link of the Rules from above.

Array Field used with "codified" table-start in the **inner/child** table should be the nested Array field used with "codified" table-start in the **outer/parent** table.